server/models/feedback.js

```diff
@@ -0,0 +1,66 @@
+const mongoose = require("mongoose");
+const Feedback = new mongoose.Schema(
```

Comment:
Just a discussion item here, but have you considered whether you would want the feedback to be nested in the user, like a [subdocument](https://mongoosejs.com/docs/subdocs.html)? I bring this up considering how usually we'd have a user wanting to view their feedback, and in that case the feedback will already be in the user schema. I may be missing some details though so feel free to discuss as a team and with Jess.

Reply:
We talk about that this morning and we decided to keep it as it is for now. The route which gets the list of feedback is protected, allowing us to filter the feedback according to the logged-in user. The list also has been modified to send only relevant data.